src/aap_eda/core/validators.py

```diff
@@ -58,6 +54,24 @@ def check_if_de_exists(decision_environment_id: int) -> int:
     return decision_environment_id
 
 
+def check_if_de_valid(image_url: str, eda_credential_id: int):
+    credential = get_credential_if_exists(eda_credential_id)
+    inputs = yaml.safe_load(credential.inputs.get_secret_value())
+    host = inputs.get("host")
```

Comment:
@hsong-rh What happens if the "host" key is missing, do you want to use empty string if its missing.

Reply:
I changed to raise an exception if `hosts` is null